backend/src/activitypub/routes/webfinger.spec.js

```diff
@@ -98,12 +98,12 @@ describe('webfinger', () => {
           expect(json).toHaveBeenCalledWith({
             links: [
               {
-                href: 'http://localhost:3000/activitypub/users/some-user',
+                href: 'http://webapp:3000/activitypub/users/some-user',
```

Comment:
I changed the no-env-warning to:

```
'WARN: No `.env` file found in `/app` (docker) or `/backend` (no docker)'
```

on the frontend this should not be displayed - i cannot find that in the code.

Reply:
This is me outsourcing the problem to you. There is a hard coded string here, and I changed it so the dockers don't have the problem. But I didn't figure out how to wildcard the value or use the config value defining the correct address.
I'd like to merge this since it is working on docker <3